diff --git a/src/parser/expression/columnref_expression.cpp b/src/parser/expression/columnref_expression.cpp
index 7afaff5d7..c1ea54dea 100644
--- a/src/parser/expression/columnref_expression.cpp
+++ b/src/parser/expression/columnref_expression.cpp
@@ -39,8 +39,8 @@ const string &ColumnRefExpression::GetTableName() const {
 }
 
 string ColumnRefExpression::GetName() const {
-	return !alias.empty() ? alias : column_names.back();
-	// return ToString();
+	// return !alias.empty() ? alias : column_names.back();
+	return ToString();
 }
 
 string ColumnRefExpression::ToString() const {
@@ -49,8 +49,8 @@ string ColumnRefExpression::ToString() const {
 		if (i > 0) {
 			result += ".";
 		}
-		result += KeywordHelper::WriteOptionallyQuoted(column_names[i]);
-		// result += column_names[i];
+		// result += KeywordHelper::WriteOptionallyQuoted(column_names[i]);
+		result += column_names[i];
 	}
 	return result;
 }
